templates/zerver/help/zulip-on-linux.md

```diff
@@ -29,10 +32,13 @@ address and a password, or using your Google or GitHub account.
 * If you decide to log in with an email/password combination, enter your email
 address in the **Email** field and your password in the **Password** field,
 then press **Sign in**.
-* If you prefer to log in with your Google or GitHub account,then press the **Sign in with Google** or **Sign in with GitHub** button.
+* If you prefer to log in with your Google or GitHub account,
+then press the **Sign in with Google** or **Sign in with GitHub** button.
 
     ![Login form](/static/images/help/linux-log-in.png)
 
 ### Adding another server
 
-To add another organization or server that you're part of, select (+) icon from the left sidebar and you can add the URL same way you did before.
+To add another organization or server that you're part of,
+select (+) icon from the left sidebar
+and you can add the URL same way you did before.
```

Comment:
@ArchitaDesai Can you also document about **updating app**?

* First set up the Zulip Desktop apt repository and its signing key, from a terminal:
```sh
sudo apt-key adv --keyserver pool.sks-keyservers.net --recv 69AD12704E71A4803DCA3A682424BE5AE9BD10D9
echo "deb https://dl.bintray.com/zulip/debian/ stable main" | \
  sudo tee -a /etc/apt/sources.list.d/zulip.list
```
* The app will be updated automatically to future versions when you do a regular software update on your system, e.g. with - 
```sh
sudo apt update && sudo apt upgrade
```

Reply:
@akashnimare Yes, I thought of mentioning how the users can set up Zulip Desktop apt repository and its signing keys, can directly download from the terminal and then the app will be updated along with the regular software update. But as I had __mentioned a link__  on the 6th line [here](https://github.com/zulip/zulip/pull/6825/files#diff-147fb43a9b038e38732380282ecdf389), I thought it would be redundant to mention it again in this document.
![image](https://user-images.githubusercontent.com/18202433/31128412-5705aed0-a86f-11e7-8bee-952f3d81b6f9.png)

Shoud I still mention that in this document?

Reply:
I think it would be nice to have those docs here as well.

Reply:
Sure, I'll do it.

Reply:
I've now added download instructions to the document